bin/packages/build.js

```diff
@@ -126,8 +126,8 @@ if ( files.length ) {
 		`${ PACKAGES_DIR }/*/src/*.scss`,
 	], {
 		ignore: [
-			`**/test/**`,
-			`**/__mocks__/**`,
+			`**/{__mocks__,__tests__,test}/**`,
```

Comment:
Where do we use `__tests__`? We should stick to one for consistency if possible.